website/content/en/docs/tasks/operator.md

```diff
@@ -95,10 +95,11 @@ The `deploy peer` command has the following flags:
 - **dataplane-replicas:** This field determines the number of ClusterLink dataplane replicas. By default, it uses 1.
 - **ingress:** This field determines the type of ingress service to expose ClusterLink deployment,
   with supported values: `LoadBalancer`, `NodePort`, or `None`. By default, it uses `LoadBalancer`.
```

Comment:
nit: is `None` useful and we should keep it? Especially since the default is LB...

Reply:
For now, let's keep him